bundles/org.openhab.binding.gpio/build.properties

```diff
@@ -0,0 +1,9 @@
+source.. = src/main/java/
```

Comment:
This `build.properties` file can be removed. It's related to the old build system.